deploy/docker/cp-tools/base/rstudio/post_commit.sh

```diff
@@ -0,0 +1,18 @@
+# Copyright 2017-2019 EPAM Systems, Inc. (https://www.epam.com/)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/usr/bin/env bash
+
+owner_home_path=`cat /etc/cp_env.sh | grep OWNER_HOME | sed 's/.*\"\([^]]*\)\".*/\1/g'`
+rm -rf $owner_home_path/.rstudio/sessions/active/session-*
```

Comment:
1. Any reason not to do like shown below? As it is a `post-hook`, which is run in the temporary container, it shall not be a big deal to setup environment variables for a current shell. Or am I missing something?
```
source /etc/cp_env.sh
rm -rf $OWNER_HOME/.rstudio/sessions/active/session-*
```

2. Another point is - what will happen if `/etc/cp_env.sh` is missing? I suggest to run the following in this case:
```
rm -rf /home/*/.rstudio/sessions/active/session-*
```
Does it make sense?